ynput/ayon-core
# Chore: Move resolve client code

## Changelog Description
Moved resolve client code next to server codebase.

## Testing notes:
1. Create package of resolve and core.
2. Upload both to server.
3. Make sure AYON launcher is using client code from this PR (e.g using dev paths).
4. Resolve integration should work as before.
